builder/common/run_config.go

```diff
@@ -402,6 +402,12 @@ type RunConfig struct {
 	//   criteria provided in `source_ami_filter`; this pins the AMI returned by the
 	//   filter, but will cause Packer to fail if the `source_ami` does not exist.
 	SourceAmiFilter AmiFilterOptions `mapstructure:"source_ami_filter" required:"false"`
+	// One of  `price-capacity-optimized`, `capacity-optimized`, `diversified` or `lowest-price`.
+	// 	 The strategy that determines how to allocate the target Spot Instance capacity
+	// 	 across the Spot Instance pools specified by the EC2 Fleet launch configuration.
+	// 	 If this option is not set/set to invalid value, Packer will use default option provided by the SDK (currently `lowest-price`).
+	// 	 For more information, see [Amazon EC2 User Guide] (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-fleet-allocation-strategy.html)
```

Comment:
I suggest removing the superfluous tabs in this section in order to be uniform vs. the other sections documented sections

Reply:
Fixed in e2ed97d.